variables.tf

```diff
@@ -26,49 +26,48 @@ variable ssh_key_name {
 
 variable zone_id {
   description = "Route53 Zone ID"
-  default = ""
+  default     = ""
 }
 
 variable host_name {
   description = "DNS host name"
-  default = ""
-}
-
-variable quay_username {
-  description = "Quay username"
-  default = "dontspamus"
-}
-
-variable quay_password {
-  description = "Quay password"
-}
-
-variable init_script {
-  description = "bash code executed before `docker-compose up -d` is called, example: `\"${file(\"init.sh\")}\"`"
-  default = ""
+  default     = ""
 }
 
 variable in_open_ports {
-  type = "list"
+  type    = list
   default = []
 }
 
 variable in_cidr_blocks {
-  type = "list"
+  type    = list
   default = ["0.0.0.0/0"]
 }
 
 variable in_source_security_group {
   description = "Security group to receive SSH access"
-  type = "string"
-  default = ""
+  type        = string
+  default     = ""
 }
 
 variable disk_utilization_alarm_threshold {
-  description = "disk occupation alarm threshold (% of disk utilization)"
-  default = "80"
+  description = "Disk occupation alarm threshold (% of disk utilization), for example 80.\nIf not set, the alarm won't be created"
+  type        = number
+  default     = 80
+}
+
+variable sns_alarm_enabled {
+  description = "Determine if the cloudwatch alarm will be forwarded to the SNS topic provided or not"
+  type        = bool
+  default     = true
+}
+
+variable sns_topic_alarm_arn {
+  type    = string
+  default = "arn:aws:sns:eu-west-1:309416224681:bellosguardo"
```

Comment:
A few remarks regarding these variables and their usage as they all fit under a common "disk utilization alarm" theme:
* the boolean variable (here `sns_alarm_enabled`) should drive the creation (or not) of the `aws_cloudwatch_alarm` resource, not the configuration of its actions;
* the disk utilization threshold should not be piggybacked to drive the creation of the alarm;
* if we move from strictly characterize `sns_topic_alarm_arn` to handle only SNS to a broader name handling the actions on alarm state we could expose a nicer and more configurable interface.

Reply:
Renamed `sns_topic_alarm_arn` to `alarm_actions`, and changed type from string to list(string)